vendors/espressif/boards/esp32/ports/ble/nimble/iot_ble_hal_gatt_server.c

```diff
@@ -38,6 +38,7 @@
 #include "bt_hal_manager.h"
 #include "bt_hal_gatt_server.h"
 #include "iot_ble_hal_internals.h"
+#include "iot_ble_config.h"
```

Comment:
Config file include before other includes?

Reply:
It is fine, the other includes are low level. But for consistency sake, I ll move it up